third_party/src/default/assets/js/main.js

```diff
@@ -431,21 +431,26 @@ function renderSimpleHTMLRecursive(obj, package, spacing) {
             if (window.location.href.indexOf('modules') == -1) {
                 href = 'modules/';
             }
-            if (window.location.href.indexOf('interfaces') > -1 ||
-                window.location.href.indexOf('assets') > -1 ||
-                window.location.href.indexOf('classes') > -1) {
+            if (window.location.href.indexOf('assets') > -1 ||
+                window.location.href.indexOf('classes') > -1 ||
+                window.location.href.indexOf('enums') > -1 ||
+                window.location.href.indexOf('interfaces') > -1
+            ) {
                 href = '../modules/';
             }
+            if(window.location.href.indexOf('modules') > -1){
+                href = "../modules/" + href;
+            }
             if (package) {
                 if (key === 'Overview') {
-                    href += package.substr(1) + ".html";
+                    href += "_" + package.substr(1) + "_.html";
```

Comment:
We can use string templating here.

Reply:
Sorry, I didn't understand all the code, I just found it here through Debugger. . .

Reply:
Oh you're right, nevermind.